app/views/articles/index.html.erb

```diff
@@ -6,4 +6,6 @@
     <h2><%= link_to(a.title, article_path(a)) %></h2>
     <p><%= a.body.length > 200 ? a.body.first(200) + "..." : a.body.first(200) %>&nbsp;<%= link_to("Read More", article_path(a)) %></p>
```

Comment:
ActiveSupport has a built-in truncate method that may be helpful here 
https://www.rubydoc.info/gems/activesupport/String:truncate